### PR TITLE
v6.0.0-alpha.6

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,58 @@
...
+
+### `@mui/x-date-pickers@v__PICKERS_VERSION__` / `@mui/x-date-pickers-pro@v__PICKERS_VERSION__`
+
+#### Changes
+
+- [DateRangePicker] Fix input focused style and mobile behaviour (#6645) @LukasTy
+- [fields] Update sections when the locale changes (#6649) @flaviendelangle
+- [pickers] Add Ukrainian (uk-UA) locale on the date picker (#6661) @Dufran
+- [pickers] Allow non-controlled usage of the calendar components (#6643) @flaviendelangle
+- [pickers] Export other adapters derived from moment or date-fns (#6571) @alexfauquette
```
Comment (flaviendelangle) [inline CHANGELOG.md:47]: I propose to add a warning when using an adapter directly from date-io in the next release and to advertise about this change then, not this week

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,58 @@
...
+
+- 🌍 Add Ukrainian (uk-UA) locale to pickers (#6661) @Dufran
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v6.0.0-alpha.6` / `@mui/x-data-grid-pro@v6.0.0-alpha.6` / `@mui/x-data-grid-premium@v6.0.0-alpha.6`
+
+#### Breaking changes
+
+- The `disableIgnoreModificationsIfProcessingProps` prop was removed and its behavior when `true` was incorporated as the default behavior.
```
Comment (alexfauquette) [inline CHANGELOG.md:28]:
```suggestion
- The `disableIgnoreModificationsIfProcessingProps` prop has been removed and its behavior when `true` is incorporated as the default behavior.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,58 @@
...
+- [DataGrid] Remove `disableIgnoreModificationsIfProcessingProps` prop (#6640) @m4theushw
+- [DataGrid] Separate private and public `apiRef` properties (#6388) @cherniavskii
+
+### `@mui/x-date-pickers@v6.0.0-alpha.6` / `@mui/x-date-pickers-pro@v6.0.0-alpha.6`
+
+#### Changes
+
+- [DateRangePicker] Fix input focused style and mobile behaviour (#6645) @LukasTy
+- [fields] Update sections when the locale changes (#6649) @flaviendelangle
+- [pickers] Add Ukrainian (uk-UA) locale on the date picker (#6661) @Dufran
```
Comment (LukasTy) [inline CHANGELOG.md:45]: 🙈 
```suggestion
- [pickers] Add Ukrainian (uk-UA) locale (#6661) @Dufran
```

File: packages/x-license-pro/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.0.0-alpha.5",
+  "version": "6.0.0-alpha.6",
```
Comment (LukasTy) [inline packages/x-license-pro/package.json:3]: Was there any license related changes that a new version was triggered?? 🤔

**Review by LukasTy — APPROVED** [review]

**Review by cherniavskii — APPROVED** [review]

**Pull request merged**